Minitwit_BE/Minitwit_BE.Api/Controllers/FollowerController.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.AspNetCore.Mvc;
+using Minitwit_BE.Domain;
+using Minitwit_BE.Persistence;
+
+namespace Minitwit_BE.Api.Controllers
+{
+    [ApiController]
+    [Route("api/followers")]
+    public class FollowerController : ControllerBase
+    {
+        private readonly TwitContext _twitContext;          // To dependency inject the context instance.
+        
+        public FollowerController(TwitContext twitContext)
+        {
+            _twitContext = twitContext;
+        }
+
+        [HttpGet("list/{id}")]
+        public async Task<ActionResult<List<Follower>>> getFollowedUsers(int id)
+        {
+            List <Follower> followersList = _twitContext.Followers.Where(entry => entry.WhoId == id).ToList();
```

Comment:
In general, it's better to use `.Equals()` method of `entry.WhoId` instead of `==`, because in case of objects you would compare the value and not the references